core/src/avm2/bytearray.rs

```diff
@@ -132,6 +132,16 @@ impl ByteArrayStorage {
         Ok(bytes)
     }
 
+    /// Same as `read_bytes`, but cuts the result at the first null byte to recreate a bug in FP.
+    pub fn read_bytes_null_terminated(&self, amnt: usize) -> Result<&[u8], EofError> {
```

Comment:
Maybe rename this to `read_utf_bytes()`, to match avmplus:
https://github.com/adobe/avmplus/blob/master/core/ByteArrayGlue.cpp#L2640

Reply:
I had no idea they hardcoded that